server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -98,7 +98,8 @@ public final DivTag render(ApplicantQuestionRendererParams params) {
 
     ImmutableList<DomContent> questionTextDoms =
         TextFormatter.createLinksAndEscapeText(
-            question.getQuestionText(), TextFormatter.UrlOpenAction.NewTab);
+            !question.isOptional() ? question.getQuestionText() + "*" : question.getQuestionText(),
```

Comment:
Suggestion: swap the order so the assertion is positive which aids readability.  isOptional ?   :